src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java

```diff
@@ -517,9 +502,35 @@ public void update() {
 						}
 						break;
 					}
-
+				} else if (godPotionMatcher.matches()) {
+					foundGodPotText = true;
+					godpotRemainingTime = Integer.parseInt(godPotionMatcher.group(1));
+					godpotRemainingTimeType = godPotionMatcher.group(2);
+				}
+				if (godpotRemainingTimeType != null) {
+					switch (godpotRemainingTimeType) {
+						case "Days":
+						case "Day":
+						case "d":
+							godPotDuration += godpotRemainingTime * 24 * 60 * 60 * 1000;
+							break;
+						case "Hours":
+						case "Hour":
+						case "h":
+							godPotDuration += godpotRemainingTime * 60 * 60 * 1000;
+							break;
+						case "Minutes":
+						case "Minute":
+						case "m":
+							godPotDuration += godpotRemainingTime * 60 * 1000;
+							break;
+						case "Seconds":
+						case "Second":
+						case "s":
+							godPotDuration += godpotRemainingTime * 1000;
+							break;
+					}
```

Comment:
We already have another method (getMinutesRemaining) doing practically the same. I think we should join them.